nginx/nginx.conf

```
@@ -3,6 +3,8 @@ server {
     listen  [::]:80;
     server_name  _;
 
+    add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
```

One more place to remove please @JGiter 
```suggestion
    add_header Strict-Transport-Security "max-age=31536000;" always;
```